packages/farcaster-js-hub-rest/src/openapi/spec.yaml

```diff
@@ -1032,8 +1032,11 @@ components:
           format: uri
         castId:
           $ref: '#/components/schemas/CastId'
-      required:
-        - url
+      oneOf:
+        - required:
+            - url
+        - required:
+            - castId
```

Comment:
Neat! Didn't know this was supported by OpenAPI v2 spec. It made the generated docs confused though (but I don't know if it's worth bending over backward for the doc generator). Also it technically made the non-required one optional. This *really* isn't important at all. If it's trivial to make them mutually exclusive, then go for it. Otherwise this is perfectly fine as-is

<img width="546" alt="Screenshot 2024-02-14 at 12 33 41 PM" src="https://github.com/standard-crypto/farcaster-js/assets/2490863/96a2451c-de5e-4b92-af7d-9bf7117dd68a">